docs/concepts/dotnet-standard-library.md

```diff
@@ -0,0 +1,151 @@
+.NET Standard Library
+=====================
+
+The .NET Standard Library is a formal specification of .NET APIs that are intended to be available on all .NET runtimes. The motivation behind the Standard Library is establishing greater uniformity in the .NET ecosystem. [ECMA 335](https://github.com/dotnet/coreclr/blob/master/Documentation/project-docs/dotnet-standards.md) continues to establish uniformity for .NET runtime behavior, but there is no similar spec for the .NET Base Class Libraries (BCL) for .NET library implementations. The .NET Standard Library is a standardized BCL, enabling developers to produce libraries that are usable across .NET app models and supported operating systems while having access to all the APIs they expect to use. 
+
+Developers use the .NET Standard Library in order to build libraries that are portable across .NET runtimes and operating systems. It can be thought of as the next generation of Portable Class Libraries (PCL). 
+
+The spec is called "The .NET Standard Library" or can be referred to as ".NET Standard" as a short form. A Library that targets the spec can be called "a .NET Standard Library" or an NSL, similar to PCL.
+
+Specification
+=============
+
+The .NET Standard Library spec is a standardized set of APIs, contained and described by packages. The spec is not singular, but an incrementally growing and linearly versioned set of APIs. The first version of the standard establishes a baseline of APIs. Subsequent versions add APIs. There is no established provision for removing APIs from the standard.
+
+The spec is maintained by .NET runtime implementors, specifically Microsoft (includes .NET Framework, .NET Core and Mono) and Unity. A public feedback process is used as part of establishing new .NET Standard Library versions.
+
+Official Artifacts
+------------------
+
+The official spec is a set of .cs files that describe the shape of the standard. Additional derivative artifacts are provided to enable more convenient reading and to enable certain developer scenarios (for example, using a compiler).
+
+- [.cs files](https://github.com/dotnet/corefx/tree/master/src)
```

Comment:
@tdykstra  I believe that the source in the `ref` directory for each package describes the .Net Standard, while the `src` directory contains the .Net Core implementation. It might make sense to explain that here.